package/Dockerfile

```diff
@@ -7,7 +7,8 @@ RUN apt-get update \
     && apt-get install -y \
     git
 
-RUN git clone -b v0.3.1 --depth 1 https://github.com/aquasecurity/kube-bench.git
+RUN git clone https://github.com/aquasecurity/kube-bench.git	
+RUN cd /kube-bench && git checkout f8e0171
```

Comment:
is there a published release-timeline about when the upstream will be tagged?

Reply:
@prachidamle There is not a published release-timeline for the upstream kube-bench yet. But we have asked a question on kube-bench [discussion forum](https://github.com/aquasecurity/kube-bench/discussions/919#discussioncomment-1858329) and we are still expecting a response.

Reply:
Updated Dockerfile with latest upstream aquasecurity/kube-bench:v0.6.6

Reply:
Thank you for updating with the upstream!